mne/epochs.py

```diff
@@ -1166,6 +1169,7 @@ def _compute_aggregate(self, picks, mode="mean"):
                 )
             data = np.zeros((n_channels, n_times))
             n_events = 0
+            # __iter__ here??
             for e in self:
```

Comment:
Yes, this is the sort of place where I think you'd want
```suggestion
            # __iter__ here??
            for e in self.__iter__(raise_error=True):
```